third_party/babel/custom-babel-helpers.js

```diff
@@ -0,0 +1,88 @@
+(function (global) {
```

Comment:
good question. nope, didn't make any changes to the file. I actually filed a [bug](https://github.com/babel/babelify/issues/123#issuecomment-149779312) to babelify that got fixed asap so I could do this (build the file in the stream), BUT the major issue is that our build is actually composed of multiple builds (`build` and `dist` tasks for example call `buildExtensions` which then call individual `buildExtension`) which i would need to gather the meta information from, and this is still doable but might have some async issues. The other way is to scan through all the files to get the meta information which might slow down the build step. Building the file in the stream is actually more ideal since if anybody introduces new features, it would be guaranteed that the helper will be there when its needed.

I decided to do this step (build offline more or less), until i finish refactoring the gulpfile and move them into their individual tasks. I'm perfectly ok with holding off on merging this until that work is done as I feel it is more safer.